lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -699,6 +788,96 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
+                        expect(cohortsView.$('input[name="cohort-assignment-type"]:checked').val()).toBe(MOCK_RANDOM_ASSIGNMENT);
+                        expect(cohortsView.$('.radio-yes').prop('checked')).toBeTruthy();
+
+                        // Update existing cohort values
+                        cohortsView.$('.cohort-name').val(newCohortName);
+                        cohortsView.$('.type-manual').prop('checked', true).change();
+                        clearContentGroup();
+
+                        // Save the updated settings
+                        cohortsView.$('.action-save').click();
+                        AjaxHelpers.expectJsonRequest(
+                            requests, 'PATCH', '/mock_service/cohorts/1',
+                            expectedRequest(MOCK_MANUAL_ASSIGNMENT)
+                        );
+                        AjaxHelpers.respondWithJson(
+                            requests,
+                            createMockCohort(newCohortName, 1, 0, null, null)
+                        );
+
+                        // Verify the new/updated cohort values
+                        expect(cohortsView.$('.cohort-name').val()).toBe(newCohortName);
+                        expect(cohortsView.$('input[name="cohort-assignment-type"]:checked').val()).toBe(MOCK_MANUAL_ASSIGNMENT);
+                        expect(cohortsView.$('.radio-no').prop('checked')).toBeTruthy();
+
+                        verifyMessage('Saved cohort', 'confirmation');
+
+                        // Now try to update existing cohort name with an empty name
+                        // We can't save a cohort with empty name, so we should see an error message
+                        cohortsView.$('.cohort-name').val('');
+
+                        saveFormAndExpectErrors('update', ['You must specify a name for the cohort']);
+                    });
+
+                    it("assignment settings are disabled for default cohort", function() {
+                        createCohortsView(this, {
+                            cohorts: [
+                                {
+                                    id: 1,
+                                    name: 'Cohort.me',
+                                    assignment_type: MOCK_RANDOM_ASSIGNMENT,
+                                    group_id: 111,
+                                    user_partition_id: MOCK_COHORTED_USER_PARTITION_ID
+                                }
+                            ],
+                            selectCohort: 1
+                        });
+
+                        // We have a single random cohort so we should not be allowed to change it assignment type
+                        expect(cohortsView.$('.cohort-management-assignment-type-settings')).toHaveClass('is-disabled');
+                        expect(cohortsView.$('.copy-error').text()).toContain("There must be one cohort to which students can be randomly assigned.");
```

Comment:
Please change "randomly" to "automatically" and move "be" before "assigned": "There must be one cohort to which students can automatically be assigned."